include/aspect/simulator_signals.h

```diff
@@ -151,6 +151,28 @@ namespace aspect
     boost::signals2::signal<void (typename parallel::distributed::Triangulation<dim> &)>  post_refinement_load_user_data;
 
     /**
+     * A signal that is called before the computation of tangential boundary
+    * conditions for which normal vectors are needed.
+    *
```

Comment:
Yes, astyle doesn't indent comments :-( You have to do it by hand, unfortunately.